### PR TITLE
Added the addition operation  of contributions

### PR DESCRIPTION
**Pull request by ashishnamdev09-09**
Made the changes related to the dashboard page .Early it was not adding the total fees of  particular event which was selected.

### CODE REVIEW

**Comment by ashishnamdev09-09** [comment]
Problem solved

File: checkout.php [hunk 1]
```diff
@@ -80,7 +80,22 @@
                         <div class="media align-items-center">
                             <span class="text-secondary">Total</span>
                             <div class="media-body text-right">
-                                <span class="font-weight-semi-bold">&#8377;<?php echo $total_amount; ?></span>
+                                <span class="font-weight-semi-bold">&#8377;<?php echo $total_amount;
+                                  //ADDITION OPERATION
+                                include 'includes/db_connect.php';
+
```
Comment (sidsbrmnn) [inline checkout.php:86]: This logic needs to go up where the registration is being inserted into the table.

**Comment by sidsbrmnn** [comment]
This fix seems a little unnecessary as I'm migrating this app to Laravel.

**Comment by ashishnamdev09-09** [comment]
ok that's great !!

**Pull request closed without merging**